Automattic/jetpack
# Components: Add PricingTable component

This PR adds a new Pricing Table component, that can be used to compare multiple plans.

#### Changes proposed in this Pull Request:

* Add new Pricing Table component.

#### Other information:


#### Jetpack product discussion


p1659435614747009-slack-C02TQF5VAJD
p6TEKc-6td-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to Storybook, and make sure the new Pricing Table component looks like the screenshot below.
* Run `npm run test` in the component package and make sure the tests pass.

<img width="996" alt="CleanShot 2022-08-03 at 16 39 52@2x" src="https://user-images.githubusercontent.com/1713699/182636496-490027ac-1fcc-4a0f-bb86-38761988829c.png">
